api/v1beta1/spiaccesstoken_types.go

```diff
@@ -125,6 +126,7 @@ const (
 	PermissionAreaRepositoryMetadata PermissionArea = "repositoryMetadata"
 	PermissionAreaWebhooks           PermissionArea = "webhooks"
 	PermissionAreaUser               PermissionArea = "user"
+	PermissionAreaRegistry           PermissionArea = "registry"
```

Comment:
Is `registry` an image registry?

I think we should be careful about this, because now quay uses `repository` as the permission area, but with `registry` added, it should maybe either use both interchangeably or switch to `registry`.

Reply:
I agree.

Reply:
Issue created: https://issues.redhat.com/browse/SVPI-248